nacos/uid.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2023 CloudWeGo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package nacos
+
+import "go.uber.org/atomic"
+
+var uniqueID atomic.Int64
```

Comment:
`atomic.Int64` is not available until go1.19, so you might need to implement one yourself

Reply:
I use "go.uber.org/atomic" other than "sync/atomic"~